package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "eslint-config-nhds",
+  "name": "lint-config-nhds",
```

Comment:
This is going to be a breaking change for ESLint: http://eslint.org/docs/developer-guide/shareable-configs (the package needs to be `eslint-config-{name}`. I think we should probably go with sub-packages in this repo.